tests/integrationtest/t/globalindex/ddl.test

```diff
@@ -4,41 +4,13 @@ create table t (a int, b int, unique index idx(a) global);
 create table t (a int, b int, index idx(a) global);
 create table t (a int, b int, unique index idx(a) global) partition by hash(b) partitions 3;
 drop table t;
--- error 8200
 create table t (a int, b int, index idx(a) global) partition by hash(b) partitions 3;
+drop table t;
 create table t3(a int not null, b int, primary key(a) nonclustered, unique idx_b(b) global) partition by hash(a) partitions 3;
 drop table t3;
 -- error 8264
 create table t (a int primary key nonclustered, b int) partition by hash(b) partitions 3;
-create table t (a int, b int, unique key (a)) partition by hash(a) partitions 3;
--- error 8264
-alter table t partition by hash(b) partitions 3;
-alter table t partition by hash(b) partitions 3 update indexes (a global);
--- error 8200
-alter table t add index idxErr (b) global;
-alter table t add unique index idxOK (b) global;
--- error 8200
-create index idxErr on t (b) global;
-create unique index idxOK2 on t (b) global;
-alter table t remove partitioning;
--- error 8200
-alter table t add index idxErr (b) global;
--- error 8200
-alter table t add unique index idxErr (b) global;
--- error 8200
-create index idxErr on t (b) global;
--- error 8200
-create unique index idxErr on t (b) global;
-drop table t;
 
```

Comment:
Why remove all of these? The added ones below does not include repartitioning, `alter table t partition by hash(b) partitions 3 update indexes (a global);` for example.

Reply:
I think they are duplicate with pervious.